CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs

```diff
@@ -39,5 +46,17 @@ public GetLateDevelopersResponse Execute()
             }
             return getLateDevelopersResponse;
         }
+        
+        private static DateTimeOffset GetStartingDate(DateTimeOffset currentDateTime)
+        {
+            var diff = (7 + (currentDateTime.DayOfWeek - DayOfWeek.Monday)) % 7;
```

Comment:
may want to give this variable a better name

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/23/commits/207c14dae404a1acbb90e06e0fee1b81388abc26.